demo/src/test/java/com/example/test/core/configs.properties

```diff
@@ -1,4 +1,4 @@
 browser=CHROME
-url=https://www.saucedemo.com/
+url=https://staging.kbb.com/whats-my-car-worth/

```

Comment:
url should be effect for all test, all team. should change to specific name

Reply:
Renamed url to urlforwhatmycarworth